README.md

```diff
@@ -276,6 +276,7 @@ A selection of major game studios, publishers, etc. using GitHub:
 * [Monster 1](https://github.com/Nooskewl/monster) - Classic-style J-RPG, prequel to Monster RPG 2.
 * [Monster RPG 2](https://github.com/Nooskewl/monster-rpg-2) - Turn-based J-RPG in a fantasy universe.
 * [Open MW](https://github.com/OpenMW/openmw) - Reimplementation of the game Morrowind.
+* [The Legend of Zelda: Mystery of Solarus DX](https://github.com/christopho/zsdx) - sequel to The Legend of Zelda: A Link to the Past released on the SNES, written in lua and created with the [Solarus] (https://github.com/christopho/solarus) engine
```

Comment:
Awesome - thanks! Solarus looks like a great addition to the list too - would you mind adding a separate entry for that under the <kbd>Frameworks/Engines/Libraries</kbd> section?